http/headers/x-content-type-options.json

```diff
@@ -49,9 +49,15 @@
             "safari_ios": {
               "version_added": false
             },
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "9.0"
+              },
+              {
+                "partial_implementation": true,
```

Comment:
Usually `partial_implementation` comes with a note explaining why it is partial support. Can you copy the one from `chrome_android` over? 

`"notes": "Not supported for stylesheets.",`